### ISSUE TITLE
Need admin.super to edit group membership

### ISSUE THREAD

**Issue by rogerhutchings** [david-szabo97/grav-plugin-admin-addon-user-manager#23]
I've created a user group called 'Admins', who I want to be able to create and manage other users. It has the following permissions set to true:

```
admin.users
admin-addon-user-manager.users
admin-addon-user-manager.groups
admin-addon-user-manager.users_expert
```

They can create new users, but they can't then add them to any groups. When trying, they get the error message `You have insufficient permissions for task save.` Setting `admin.super` will allow this, but that isn't ideal. Is there an additional permission I need to set to enable editing of group membership?

**Comment by david-szabo97** [comment]
The user editor is part of the **Admin** plugin, currently there is no permission which allows you to change the permissions of an user other than the `admin.super`.
I will look into the internals to find a way to make this happen.

**Comment by rogerhutchings** [comment]
Got it. I really appreciate you having a look 👍

**Comment by david-szabo97** [comment]
Fixed in v1.9.0 (https://github.com/david-szabo97/grav-plugin-admin-addon-user-manager/commit/f3edba2609ced7bbd825a348f0e9c1a0a426f01d)

**Comment by rogerhutchings** [comment]
Awesome, thank you @david-szabo97
